ostd/src/mm/frame/meta.rs

```diff
@@ -555,6 +555,11 @@ fn alloc_meta_frames(tot_nr_frames: usize) -> (usize, Paddr) {
 pub struct UnusableMemoryMeta;
 impl_frame_meta_for!(UnusableMemoryMeta);
 
+/// Reserved memory metadata. Maybe later used as I/O memory.
```

Comment:
The changes in this file are not useful, too.

I think it's okay to keep the changes because it generally makes sense. If you decide to keep the changes, please fix the comments around `UnusableMemoryMeta`:
```rust
/// Unusable memory metadata. Cannot be used for any purposes.
#[derive(Debug)]
pub struct UnusableMemoryMeta;
impl_frame_meta_for!(UnusableMemoryMeta);

/// Reserved memory metadata. Maybe later used as I/O memory.
#[derive(Debug)]
pub struct ReservedMemoryMeta;
impl_frame_meta_for!(ReservedMemoryMeta);
```